english/TheEpicSax/111_TheEpicSaxGuyArrives.md

```diff
@@ -1,3 +1,6 @@
 So to continue with the story, the situation was tense, but suddenly everything got solved on it's own.
 The Epic Sexy Sax Guy appeared at the scene and with his powerful skills with the sax, he calmed even the most affected 
-by the catastrophe... [Click here to receive the epic sax guy music] (https://www.youtube.com/watch?v=BYEFv6sgKrM)
\ No newline at end of file
+by the catastrophe... [Click here to receive the epic sax guy music] (https://www.youtube.com/watch?v=BYEFv6sgKrM)
```

Comment:
Not broken
El abr 17, 2016 7:06 PM, "Casey Collaborator" <notifications@github.com>
escribió:

> In english/TheEpicSax/111_TheEpicSaxGuyArrives.md
> <https://github.com/udacity/create-your-own-adventure/pull/4247#discussion_r59994196>
> :
>
> > \ No newline at end of file
> > +by the catastrophe... [Click here to receive the epic sax guy music] (https://www.youtube.com/watch?v=BYEFv6sgKrM)
>
> It looks like this link is broken. You can view the page on GitHub on your
> fork and click the link there to check if it is working correctly.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/udacity/create-your-own-adventure/pull/4247/files/c74da808b16a19ecff5817333dcf66676ccdb0e9#r59994196>
>